src/Microsoft.DotNet.Cli.Utils/CommandResolution/ProjectToolsCommandResolver.cs

```diff
@@ -141,9 +141,33 @@ private CommandSpec ResolveFromProjectTools(CommandResolverArguments commandReso
                 toolPackageFramework,
                 possiblePackageRoots);
 
+
+            //  NuGet restore in Visual Studio may restore for netcoreapp1.0.  So if that happens, fall back to
+            //  looking for a netcoreapp1.0 or netcoreapp1.1 tool restore.
             if (toolLockFile == null)
             {
-                return null;
+                if (toolPackageFramework.Framework == FrameworkConstants.FrameworkIdentifiers.NetCoreApp &&
+                    toolPackageFramework.Version >= new Version(2, 0, 0))
+                {
+                    foreach (var fallbackFramework in new [] { "netcoreapp1.1", "netcoreapp1.0"})
```

Comment:
Should we just add project.DotnetCliToolTargetFramework to this array and remove the GetToolLockFile call above?